python/tk_desktop2/websockets/websockets_server.py

```diff
@@ -205,10 +207,10 @@ def _new_connection(self, socket_id, name, address, port, request):
 
         # The origin coming from the request's raw header will be a bytearray. We're
         # going to want it as a string, so we'll go ahead and convert it right away.
-        origin_site = str(request.rawHeader("origin"))
+        origin_site = request.rawHeader(QtCore.QByteArray(six.b("origin"))).data()
```

Comment:
Creating a QByteArray requires bytes and on Python3, we were passing a string.

Reply:
`b"origin"` would be enough here.